src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -378,11 +384,29 @@ private async Task<IConfidentialClientApplication> BuildConfidentialClientApplic
             }
             catch (Exception ex)
             {
-                _logger.LogInformation(ex, "Exception acquiring token for a confidential client. ");
+                _logger.LogInformation(
+                    ex,
+                    string.Format(CultureInfo.InvariantCulture, 
+                    "Exception acquiring token for a confidential client. "));
                 throw;
             }
         }
 
+        internal void CheckApplicationOptionsContainClientSecret(ConfidentialClientApplicationOptions applicationOptions)
```

Comment:
I think that this fix is not necessary in Identity.Web, and that it should be only in MSAL. But I can understand having a tactical fix in Identity.Web if you cannot wait for the next MSAL release (or you can hotfix MSAL).

Reply:
We want to check the other parameters anyway.